profiling/Cargo.toml

```diff
@@ -27,6 +27,7 @@ uuid = { version = "1.0", features = ["v4"] }
 
 [features]
 profiling = []
```

Comment:
Ha, we should remove this `profiling` feature. It's from a very old R&D idea that apparently made it in unnoticed ^_^

I mean, we don't use that feature anywhere, right?

Reply:
Just checked, that feature is not used anywhere

Reply:
✔️